### PR TITLE
feat(ui): add peerId to settings

### LINKED ISSUE #1151: Update onboarding copy
`This is your handle — it's unique to you! Click to copy it and share with others so that they can find you` 
But we're actually sharing your peerID, and it's unique to your machine. You could have multiple peer ID's in the end when you have multiple machines.

### PR DESCRIPTION
**Pull request by juliendonck**
- added peerID to settings
- fix copy in onboarding

download

### CODE REVIEW

**Review by xla — COMMENTED** [review]
Dope and concise!

🎉 🔎 🚘 🚋

**Review by xla — APPROVED** [review]
🎉 🔎 🚘 🚋

File: ui/Screen/Onboarding/Success.svelte [hunk 1]
```diff
@@ -45,8 +45,8 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your handle — it's unique to you!<br /> Click to copy it and share
-      with others so that they can find you.
+      This is your peerID — it's unique to your machine!<br /> Click to copy it and
+      share with others so that they can add you as a remote of a project.
```
Comment (rudolfs) [inline ui/Screen/Onboarding/Success.svelte:49]: Now the copy is talking about `PeerId` but the actual thing shown in the UI is a `URN`.

**Review by rudolfs — CHANGES_REQUESTED** [review]

File: ui/Screen/Onboarding/Success.svelte (hunk 1 above)
Comment (rudolfs) [reply ui/Screen/Onboarding/Success.svelte:49]: Screenshot 2020-10-30 at 15 47 25

File: ui/Screen/Settings.svelte [hunk 2]
```diff
@@ -168,6 +169,20 @@
       </div>
     </section>
 
+    <section>
+      <header>
+        <h3>Peer ID</h3>
+      </header>
+      <div class="section-item">
+        <div class="info">
+          <p>Share your peerId with others so they can add you as a remote</p>
```
Comment (FintanH) [inline ui/Screen/Settings.svelte:178]:
```suggestion
          <p>Share your peerID with others so they can add you as a remote</p>
```

To be consistent with the welcome screen

File: ui/Screen/Settings.svelte [hunk 3]
```diff
@@ -168,6 +169,20 @@
...
+        </div>
+        <div class="action">
+          <StyledCopyable value={identity.fallback.peerId} />
```
Comment (FintanH) [inline ui/Screen/Settings.svelte:181]: What's this `fallback` thing? Should it not just be `identity.peerId`, or is this a frontend thing?

**Review by FintanH — CHANGES_REQUESTED** [review]

File: ui/Screen/Onboarding/Success.svelte (hunk 1 above)
Comment (abbey-titcomb) [reply ui/Screen/Onboarding/Success.svelte:49]: Didn't we at one time agree that we were going to refer to this as a handle? Did we decide to go back to peerID? @cloudhead @brandonhaslegs 

If so, lmk bc I have to update the glossary

File: ui/Screen/Settings.svelte (hunk 3 above)
Comment (rudolfs) [reply ui/Screen/Settings.svelte:181]: Yeah, this is wrong. It should be:
```diff
diff --git a/ui/Screen/Settings.svelte b/ui/Screen/Settings.svelte
index bc57619f..dbb938e8 100644
--- a/ui/Screen/Settings.svelte
+++ b/ui/Screen/Settings.svelte
@@ -1,5 +1,5 @@
 <script>
-  import { onMount } from "svelte";
+  import { getContext, onMount } from "svelte";

   import {
     clear,
@@ -11,7 +11,6 @@
   } from "../src/session.ts";
   import { themeOptions } from "../src/settings.ts";
   import * as path from "../src/path.ts";
-  import * as identity from "../src/identity.ts";
   import * as modal from "../src/modal.ts";
   import { getVersion, isDev } from "../../native/ipc.js";

@@ -22,6 +21,8 @@
     StyledCopyable,
   } from "../DesignSystem/Component";

+  const session = getContext("session");
+
   const updateTheme = event =>
     updateAppearance({ ...$settings.appearance, theme: event.detail });

@@ -178,7 +179,7 @@
           <p>Share your peerId with others so they can add you as a remote</p>
         </div>
         <div class="action">
-          <StyledCopyable value={identity.fallback.peerId} />
+          <StyledCopyable value={session.identity.peerId} />
         </div>
       </div>
     </section>
```

Comment (FintanH) [reply ui/Screen/Settings.svelte]: :bow: I'll be a frontend dev soon enough

File: ui/Screen/Onboarding/Success.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/Screen/Onboarding/Success.svelte]: ok undid this. But this is basically pointless right now. Cause users can't do anything with their urn... So copy is correct for the useless thing.

File: ui/Screen/Settings.svelte (hunk 2 above)
Comment (FintanH) [reply ui/Screen/Settings.svelte:179]: Should is still be called `peerID`? Or `Peer ID`? What's the glossary say?

File: ui/Screen/Settings.svelte (hunk 3 above)
Comment (rudolfs) [reply ui/Screen/Settings.svelte]: Screenshot 2020-10-30 at 16 26 01

Comment (juliendonck) [reply ui/Screen/Settings.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/1152/commits/1d6ee2ca9c6df90816d712982a44722adeefae99

**Review by FintanH — APPROVED** [review]
:kissing::ok_hand:

**Comment by rudolfs** [comment]
What about `should we make urn on profile not copyable`?
From: https://github.com/radicle-dev/radicle-upstream/issues/1136.

**Comment by juliendonck** [comment]
> What about `should we make urn on profile not copyable`?
> From: #1136.

It doesn't have the copy icon next to it, it's basically invisible that it's copyable. I don't think we'll gain anything from removing that functionality.

**Review by rudolfs — APPROVED** [review]

**Pull request merged**